cirq/work/sampler.py

```diff
@@ -199,3 +199,57 @@ async def run_sweep_async(
             An awaitable TrialResult.
         """
         return self.run_sweep(program, params=params, repetitions=repetitions)
+
+    def run_batch(
+            self,
+            programs: List['cirq.Circuit'],
+            params_list: Optional[List['cirq.Sweepable']] = None,
+            repetitions: Union[int, List[int]] = 1,
+    ) -> List[List['cirq.TrialResult']]:
+        """Runs the supplied circuits.
+
+        Each circuit provided in `programs` will pair with the optional
+        associated parameter sweep provided in the `params_list`, and be run
+        with the associated repetitions provided in `repetitions` (if
+        `repetitions` is an integer then all runs will have that number of
```

Comment:
nit: I think there should be a comma after integer.

Reply:
Done.